src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(a, b) {
+      return a + b;
+    },
+    subtract(a, b) {
+      return a - b;
```

Comment:
[CODE KNOWLEDGE] - Similar to the add method, the subtract method should only take one parameter and subtract it from the result property using the -= operator.